api/src/main/java/org/openmrs/module/kenyaemr/reporting/builder/hts/HTSRegisterReportBuilder.java

```diff
@@ -137,6 +118,7 @@ protected DataSetDefinition datasetColumns() {
         dsd.addColumn("tbScreening", new HTSTBScreeningDataDefinition(), null);
         dsd.addColumn("everHadHIVSelfTest", new HTSSelfTestDataDefinition(), null);
         dsd.addColumn("assessedForHIVRisk", new HTSRiskDoneDataDefinition(),"", new HTSRiskAssessedConverter());
+        dsd.addColumn("Linked", new HTSLinkageIdentifierDataDefinition(), null);
```

Comment:
Does this also pick CCC number from person identifier? I can't remember clearly.

Reply:
The query is getting ccc number from  a coalesce of (unique_patient_no from demographics, ccc_number from etl_hts_referral_and_linkage) .